app/src/main/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentPresenter.kt

```diff
@@ -20,6 +20,7 @@ import org.oppia.android.databinding.TopicPracticeHeaderViewBinding
 import org.oppia.android.databinding.TopicPracticeSubtopicBinding
 import org.oppia.android.domain.oppialogger.OppiaLogger
 import javax.inject.Inject
+import org.oppia.android.R
```

Comment:
Why do we need this import now?

Reply:
Must be imported accidently. Fixed it though